src/Commands/MakeUser.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Portable\FilaCms\Commands;
+
+use Spatie\Permission\Traits\HasRoles;
+use Illuminate\Console\Command;
+use Filament\Models\Contracts\FilamentUser;
+use Spatie\Permission\Models\Role;
+use Schema;
+use Hash;
+
+class MakeUser extends Command
+{
+    protected $signature = 'fila-cms:make-user {--dry-run}';
+
+    protected $description = 'Creates a new user with an admin role';
+
+    public function handle()
+    {
+        $dryRun = $this->option('dry-run');
+
+        // check if Role table exists
+        if (Schema::hasTable('role') === FALSE) {
+
+        }
+
+        // check first if there's an admin role in place
+        $checkRole = Role::where('name', 'Admin')->first();
+
+        if (is_null($checkRole) === TRUE) { // no role, prompt the user to install
+            $this->error('Fila CMS not detected');
```

Comment:
I'd be more specific here.  Clearly they have installed the package (because that's where the command is running), but haven't run the install command.

So maybe "Can't locate Admin role.  Have you run artisan fila-cms:install?"

And then return.  Don't continue.  OR we could ask the user "Do you want to run this command now?", then run the command and continue.  Or bail if they say no.

Reply:
Got it.  I think I'll go with the latter. If they execute the installation from this command, then they might miss adding arguments that are relevant to what they are doing.